src/utils/countryFeatures.ts

```diff
@@ -17,7 +17,7 @@ export function getCountryFeatures(countryCodeAlpha2: string | null): SpecificCo
 }
 
 export const getCountryFeaturesSelector = createSelector(
-  userLocationDataSelector,
+  (state: RootState) => state.networkInfo.userLocationData,
```

Comment:
Weird, this shouldn't be needed 🤔 
I think the problem is we created an import cycle between `networkInfo/selectors.ts` and `utils/countryFeatures.ts`.
We should move `userInSanctionedCountrySelector` here to avoid it.

Reply:
Fixed in 100992a0441ccd9d827c9971a8703e114989b098!